### PR TITLE
fix: `strategyId`s

### PR DESCRIPTION
**Pull request by therealharpaljadeja**
`strategyId` fixed based on the `_name` constructor argument required when deploying strategy

### CODE REVIEW

**Review by codenamejason — APPROVED** [review]

File: shared/contractCall.ts [hunk 1]
```diff
@@ -63,29 +63,33 @@ export async function getStrategyContractGroup(
...
+    case "0x0362c85758aca10c506ca07c8a16e387eae5f6e3ed0bbecd62ce420863550b17":
       return {
         strategyId: strategyId,
         contractGroupName: "allov2.RFPSimpleStrategy",
         contractGroups: ["allov2.RFPSimpleStrategy"],
       };
 
-    case "0x414f2ea9b91b8ee2e35a380fa0af0e14079832cc93530a61a4893b3dbf0a9aba":
+    // RFPCommitteev1
+    case "0x77ebe7547cdda23b3167d8264ea3b591f19fd3dae29aa83630cc4ad5258a4ae5":
```
Comment (codenamejason) [inline shared/contractCall.ts:76]: RFPCommitteeStrategyv1 

b0b4e75eea76695f3b84ca21a9ed99b9ebda3e72c23f067c6a05733edf3a0c7d

File: shared/contractCall.ts [hunk 2]
```diff
@@ -63,29 +63,33 @@ export async function getStrategyContractGroup(
           "allov2.MicroGrantsCommon",
         ],
       };
-
-    case "0xb87f34c0968bd74d43a6a5b72831a5ea733a4783a026b9fc9b1d17adf51214d2":
+      
+    // RFPSimplev1
+    case "0x0362c85758aca10c506ca07c8a16e387eae5f6e3ed0bbecd62ce420863550b17":
```
Comment (codenamejason) [inline shared/contractCall.ts:68]: RFPSimpleStrategyv1

3fa024ac367bb80131cea51d4c283471df121f8a0450ff5e6385de1b63d201a4

File: shared/contractCall.ts [hunk 3]
```diff
@@ -63,29 +63,33 @@ export async function getStrategyContractGroup(
...
+    case "0x77ebe7547cdda23b3167d8264ea3b591f19fd3dae29aa83630cc4ad5258a4ae5":
       return {
         strategyId: strategyId,
         contractGroupName: "allov2.RFPCommitteeStrategy",
         contractGroups: ["allov2.RFPCommitteeStrategy"],
       };
 
-    case "0xed28ce0387d1786c1a38404047e9eecc4d1dcaeff695b867e912483e36c3d770":
+    // QVSimplev1
+    case "0x791aa1afdfa967c276345b04f603ae73c0fa8e61c0173ca01cb4a4ec7e93d091":
```
Comment (codenamejason) [inline shared/contractCall.ts:84]: QVSimpleStrategyv1

95d1fb6ac58b2183a843bd9e614ba2410fe8ae04860e8787dede29eb5f754a98

File: shared/contractCall.ts [hunk 4]
```diff
@@ -63,29 +63,33 @@ export async function getStrategyContractGroup(
...
       return {
         strategyId: strategyId,
         contractGroupName: "allov2.QVSimpleStrategy",
         contractGroups: ["allov2.QVSimpleStrategy"],
       };
 
-    case "0xc5263e972c91d7ff40708bc71239a2b6cbc8768704e210ca3069e2e11fc195df":
+    // DonationVotingMerkleDistributionDirectTransferv1
```
Comment (codenamejason) [inline shared/contractCall.ts:91]: DonationVotingMerkleDistributionDirectTransferStrategyv1

5d2e00788e74ce133352e0a0c93ac65f3ecde07fa2cfa234abba35cd86ba6b64

File: shared/contractCall.ts [hunk 5]
```diff
@@ -95,7 +99,8 @@ export async function getStrategyContractGroup(
         ],
       };
 
-    case "0xecc48557f4826bd1181a4495232d6d07f248ef9cc0a650e64520f6c9f7458a8c":
+    // DonationVotingMerkleDistributionVaultv1
+    case "0xe5627de3d087fb2486828b06b5434ee857bc25b4eab1ce67aa3ba126dfbd5313":
```
Comment (codenamejason) [inline shared/contractCall.ts:103]: DonationVotingMerkleDistributionVaultStrategyv1

5a201e6d948094c2653e04850b8e42e4bf97d6945d8befd416c80245596c9d3e

**Review by codenamejason — CHANGES_REQUESTED** [review]

**Review by codenamejason — DISMISSED** [review]

**Comment by therealharpaljadeja** [comment]
This is how I got these values

The below value already exists

Screenshot 2024-01-08 at 1 56 42 AM

https://github.com/allo-protocol/allo-v2-spec/blob/95f38ba487b62b86051a4861ebcd6221f403440b/shared/contractCall.ts#L25-L32

This value is changed in the PR

Screenshot 2024-01-08 at 1 57 34 AM

**Comment by therealharpaljadeja** [comment]
The names are suffixed with `v1.0` not `v1`, this can be verified, by calling `getStrategyId` on strategies deployed by Allo team.

However, the indexer is looking for hash value of strategy names with suffix `v1`
 
image

**Pull request closed without merging**